db/migrate/20160607141810_remove_database_synchronization_role.rb

```diff
@@ -0,0 +1,15 @@
+class RemoveDatabaseSynchronizationRole < ActiveRecord::Migration[5.0]
+  class ServerRole < ActiveRecord::Base; end
+  class MiqServer < ActiveRecord::Base; end
```

Comment:
You can create a fake class there...but I'm not sure you actually need it for the spec, unless I'm missing something